.github/workflows/soroban-rpc.yml

```diff
@@ -25,19 +29,68 @@ jobs:
       - uses: ./.github/actions/setup-go
         with:
           go-version: ${{ matrix.go }}
-      - run: go test -race -cover -timeout 25m -v ./cmd/soroban-rpc/...
+      - run: |
+          make build-libpreflight
```

Comment:
please add a step to the golangci-lint.yml as well, before running the linter.
Otherwise, the linter would not be able to compile the code and generate some nonsense errors.

Reply:
I don't think the linter needs the binary library. What makes you think that? (I didn't see any errors in the linting output)

Reply:
it printout "could not import C (cgo preprocessing failed) (typecheck)". I'm hoping that if it would be able to compile it, then it would stop complaining about that.

Reply:
Ah, I hadn't seen that.

Reply:
That fixed it